### PR TITLE
LPS-193228 Returns the name as HTML title in case of content page draft layout

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPS-193228

cc: @ZakiMiklos

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-impl/src/com/liferay/portal/model/impl/LayoutImpl.java [hunk 1]
```diff
@@ -691,6 +691,10 @@ public String getHTMLTitle(Locale locale) {
 	 */
 	@Override
 	public String getHTMLTitle(String localeLanguageId) {
+		if (isTypeContent() && isDraftLayout()) {
```
Comment (ealonso) [inline portal-impl/src/com/liferay/portal/model/impl/LayoutImpl.java:694]: @lfbesada What happen with display pages?

Comment (lfbesada) [reply portal-impl/src/com/liferay/portal/model/impl/LayoutImpl.java:694]: Display pages keep the HTML Title in the draft layout, see: https://github.com/liferay-echo/liferay-portal/commit/a9d26357
In the case of content page, this info is always held by the published layout.

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:stop

**Comment by lfbesada** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:forward

**Comment by lfbesada** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/139259

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**